src/core/fonts_utils.js

```diff
@@ -78,9 +78,12 @@ const MacStandardGlyphOrdering = [
   "threequarters", "franc", "Gbreve", "gbreve", "Idotaccent", "Scedilla",
   "scedilla", "Cacute", "cacute", "Ccaron", "ccaron", "dcroat"];
 
-function getFontType(type, subtype) {
+function getFontType(type, subtype, isStandardFont) {
```

Comment:
```suggestion
function getFontType(type, subtype, isStandardFont = false) {
```
since there's a couple of call-sites where this new third parameter isn't provided.